docs/index.rst

```diff
@@ -28,19 +28,62 @@
 
 |fury| |ossrank| |downloads| |pre-commit|
 
-Run your dbt Core projects as `Apache Airflow® <https://airflow.apache.org/>`_ DAGs and Task Groups with a few lines of code. Benefits include:
 
-- Run dbt projects against Airflow connections instead of dbt profiles
-- Native support for installing and running dbt in a virtual environment to avoid dependency conflicts with Airflow
-- Run tests immediately after a model is done to catch issues early
-- Utilize Airflow's data-aware scheduling to run models immediately after upstream ingestion
-- Turn each dbt model into a task/task group complete with retries, alerting, etc.
+Welcome to Astronomer Cosmos! Whether you're an experienced data practitioner or just getting started, Cosmos makes it
+simple to manage and orchestrate your dbt workflows using `Apache Airflow® <https://airflow.apache.org/>`_, saving you
+time and effort. By automatically turning dbt workflows into Airflow DAGs, Cosmos allows you to focus on building
+high-quality data models without the hassle of managing complex integrations.
 
+To get started right away, please check out our `Quickstart Guides <https://astronomer.github.io/astronomer-cosmos/getting_started/index.html>`_.
+You can also explore more examples in the `/dev/dags <https://github.com/astronomer/astronomer-cosmos/tree/main/dev/dags>`_
+or in the `cosmos-demo repo <https://github.com/astronomer/cosmos-demo>`_.
 
-Example Usage
-___________________
+To learn more about about Cosmos, please read on.
 
-You can render a Cosmos Airflow DAG using the ``DbtDag`` class. Here's an example with the `jaffle_shop project <https://github.com/dbt-labs/jaffle_shop>`_:
+
+What Is Astronomer Cosmos?
+___________________________
+
+Astronomer Cosmos is an open-source library that bridges Apache Airflow and dbt, allowing you to easily transform your
+dbt projects into Airflow DAGs and manage everything seamlessly. With Cosmos, you can write your data transformations
+using dbt and then schedule and orchestrate them with Airflow, making the entire process smooth and straightforward.
+
+**Why Cosmos?**  Integrating dbt and Airflow can be complex, but Cosmos simplifies it by seamlessly connecting these
+powerful tools—letting you focus on what matters most: delivering impactful data models and results without getting
+bogged down by technical challenges.
+
+
+Why Should You Use Cosmos?
+___________________________
+
+Cosmos makes orchestrating dbt workflows:
```

Comment:
```suggestion
Cosmos makes orchestrating dbt-core workflows:
```
Shall we use "Dbt-core" instead of "dbt"?

Reply:
@tatiana / @pankajkoti any thoughts ^^?

Reply:
not too sure. Maybe we could add a line somewhere that in general when we mean `dbt`, we're referring to `dbt-core` workflows throughout the references. no strong opinion.